lib/most_popular_finder.rb

```diff
@@ -0,0 +1,31 @@
+class MostPopularFinder
```

Comment:
Extracting this into a separate class is a great call. We call this a "utility" class because it's not necessarily representing a domain object like a vehicle or facility, but it's performing some utility for us. This name is fine IMO.